sympy/physics/mechanics/_system.py

```diff
@@ -0,0 +1,937 @@
+    origin : Point
+        Global origin of the system.
+    frame : ReferenceFrame
+        Inertial reference frame of the system.
+    x : Vector
+        Unit vector in the x direction of the inertial reference frame.
+    y : Vector
+        Unit vector in the y direction of the inertial reference frame.
+    z : Vector
+        Unit vector in the z direction of the inertial reference frame.
+    q : Matrix
+        Matrix of all the generalized coordinates.
+    u : Matrix
+        Matrix of all the generalized speeds.
+    q_ind : Matrix
+        Matrix of the independent generalized coordinates.
+    q_dep : Matrix
+        Matrix of the dependent generalized coordinates.
+    u_ind : Matrix
+        Matrix of the independent generalized speeds.
+    u_dep : Matrix
+        Matrix of the dependent generalized speeds.
+    kdes : Matrix
+        Matrix of the kinematic differential equations.
+    bodies : tuple of BodyBase subclasses
+        Tuple of all bodies that make up the system.
+    joints : tuple of Joint
+        Tuple of all joints that connect bodies in the system.
+    loads : tuple of LoadBase subclasses
+        Tuple of all loads that have been applied to the system.
+    holonomic_constraints : Matrix
+        Matrix with the holonomic constraints as rows.
+    nonholonomic_constraints : Matrix
+        Matrix with the nonholonomic constraints as rows.
+    eom_method : class
+        Backend for forming the equations of motion.
+
+    Examples
+    ========
+
+    In the example below a cart with a pendulum is created. The cart moves along
+    the x axis of the rail and the pendulum rotates about the z axis. The length
+    of the pendulum is ``l`` with the pendulum represented as a particle. To
+    move the cart a time dependent force ``F`` is applied to the cart.
+
+    As the ``_system.py`` module is experimental, it is not yet part of the
+    ``sympy.physics.mechanics`` namespace. ``System`` must therefore be
+    imported directly from the ``sympy.physics.mechanics._system`` module.
+
+    >>> from sympy.physics.mechanics._system import System
+
+    We first need to import some functions and create some of our variables.
+
+    >>> from sympy import symbols, simplify
+    >>> from sympy.physics.mechanics import (
+    ...     mechanics_printing, dynamicsymbols, RigidBody, Particle,
+    ...     ReferenceFrame, PrismaticJoint, PinJoint)
+    >>> mechanics_printing(pretty_print=False)
+    >>> g, l = symbols('g l')
+    >>> F = dynamicsymbols('F')
+
+    The next step is to create bodies. It is also useful to create a frame for
+    locating the particle with respect to the pin joint later on, as a particle
+    does not have a body-fixed frame.
+
+    >>> rail = RigidBody('rail')
+    >>> cart = RigidBody('cart')
+    >>> bob = Particle('bob')
+    >>> bob_frame = ReferenceFrame('bob_frame')
+
+    Initialize system, with the rail as Newtonian reference. The body is also
+    automatically added to the system.
```

Comment:
```suggestion
    Initialize the system, with the rail as the Newtonian reference. The body is also
    automatically added to the system.
```

Reply:
Addressed in 9d9a8c1cc3532150f0e4764d9c5a6016cc80a939